app/javascript/react/store/fixedStreamSlice.ts

```diff
@@ -16,7 +16,7 @@ interface FixedStreamState {
   averageMeasurementValue: number | null;
   isLoading: boolean;
 }
-
+const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
```

Comment:
I see we have a duplicate in [Graph.tsx](https://github.com/HabitatMap/AirCasting/pull/679/files#diff-7471bf256dae87ada9622e22f537149dfd03e2c54b4139c0bed600bbacfb5266)